.eslintrc

```diff
@@ -17,7 +17,8 @@
 	},
 	"rules": {
 		"jsdoc/check-indentation": "error",
-		"no-shadow": "warn"
+		"no-shadow": "warn",
+		"@wordpress/no-unsafe-wp-apis": "off" // need to remove in future when component is updated.
```

Comment:
Thought I flagged this elsewhere but strongly we don't turn this off, but suppress them inline in each place we report them so it's abundantly clear to contributors.

Or better yet, don't suppress them at all - switch them to `warn` if the default is `error` and we're blocking building.